modules/pxp-module-puppet

```diff
@@ -154,7 +156,7 @@ def metadata()
           },
           :required => [:env, :flags]
         },
-        :output => {
+        :results => {
```

Comment:
PCP-251 is about both that schema change and how to convey the output.

I think the point here is whether or not this should go in a separate commit or PR. I don't have any strong opinion on that.